src/main/java/com/google/starfish/servlets/SearchServlet.java

```diff
@@ -0,0 +1,122 @@
+package com.google.starfish.servlets;  
+
+import java.io.IOException;  
+import java.sql.Connection;  
+import java.sql.SQLException;  
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import javax.servlet.ServletException;  
+import javax.servlet.annotation.WebServlet;  
+import javax.servlet.http.HttpServlet;  
+import javax.servlet.http.HttpServletRequest;  
+import javax.servlet.http.HttpServletResponse;  
+import javax.sql.DataSource;
+import java.util.Date;
+import com.google.starfish.services.NoteService;
+import com.google.starfish.models.Note;
+import com.google.gson.Gson;
+import java.util.ArrayList;
+import java.util.List;
+
+/** Servlet that returns search results for notes based on school and course */
+@WebServlet("/search")  
+public class SearchServlet extends HttpServlet {  
+
+  private NoteService noteService = new NoteService();
+
+  @Override
+  public void doGet(HttpServletRequest req, HttpServletResponse res) throws ServletException, IOException {
+    DataSource pool = (DataSource) req.getServletContext().getAttribute("my-pool");  
+
+    String reqSchool = trimAndLowerCaseString(req.getParameter("school"));
+    String reqCourse = trimAndLowerCaseString(req.getParameter("course"));
+
+    try (Connection conn = pool.getConnection()) {
+      try {
+        conn.setAutoCommit(false);
+        String stmt = getSearchQuery(reqSchool, reqCourse);
+        try (PreparedStatement getNotesStatement = conn.prepareStatement(stmt)) {
+          getNotesStatement.setString(1, reqSchool);
+          getNotesStatement.setString(2, reqCourse);
+          ResultSet rs = getNotesStatement.executeQuery();
+          conn.commit();
+
+          List<Note> notes = createNotesFromResultOfSeach(pool, rs);
```

Comment:
I think you mean *Search not *Seach (here and in method defn below) :) 

nit: while you're at it though, I think `buildNotesForSearchResults` or `buildNotesFromQueryResult` would be a tad less verbose and a little more accurate than "create" because you're not like creating notes from scratch, you know? And also I think it would be helpful to be a little more consistent in this naming since it sounds like you're using "search" to refer to the user-facing thing, whereas "query" seems to be referring to the internal thing you're doing

Reply:
Ahhh sorry about the typo haha, and I think you're right about renaming, I like `buildNotesFromQueryResult` as this function is taking a query result set as a param.